src/services/api/plugins.ts

```diff
@@ -0,0 +1,26 @@
+/*
```

Comment:
Please move this file to `src/services/registry` as it doesn't provide any iterations with API rather fetches plugins from a registry.

In addition, create new Axios instance to avoid passing an empty authorization header. See 
https://github.com/che-incubator/che-dashboard-next/blob/2aeabaca51a73c8a121c5e94b0905db94edbe8d4/src/services/registry/devfiles.ts#L15-L16

Reply:
done